test/rpc/test_rpc_transactions.py

```diff
@@ -320,3 +323,125 @@ def test_get_deploy_transaction_receipt(deploy_info):
         "statusData": None,
         "actual_fee": "0x0"
     }
+
+
+def test_add_invoke_transaction(invoke_content):
+    """
+    Add invoke transaction
+    """
+    function_invocation = RpcInvokeTransaction(
+        contract_address=invoke_content["contract_address"],
+        entry_point_selector=invoke_content["entry_point_selector"],
+        calldata=invoke_content["calldata"],
+    )
+
+    resp = rpc_call(
+        "starknet_addInvokeTransaction",
+        params={
+            "function_invocation": function_invocation,
+            "signature": invoke_content["signature"],
+            "max_fee": hex(0),
+            "version": hex(constants.TRANSACTION_VERSION),
+        }
+    )
+    receipt = resp["result"]
+
+    assert set(receipt.keys()) == set(["transaction_hash"])
```

Comment:
Here and in other places we are doing just key assertions, no values are being asserted. This can be done either how it's done in other rpc tests, or you could use `assert_valid_schema` as it's done in `test_account_predeployed.py`

Reply:
I'm having a hard time getting expected values for these assertions.

The class hashes here are different than when computed using gateway.
Same goes for transaction hash, probably since it's missing abi in contract_definition.

Could you point me to any source for the expected results?

Reply:
yeah, we don't have to assert the precise value, but at least the structure of the response (i.e. that the value is a hex string).